### PR TITLE
Add download and time interval refresh

### PR DESCRIPTION
**Pull request by fajarutamaa**

### CODE REVIEW

File: src/views/edm/EDMBenchmarkChart.vue [hunk 1]
```diff
@@ -378,6 +385,7 @@ export default {
     }),
 
     update() {
+      console.log('Updating EDM Benchmark Chart...')
```
Comment (indrarudianto) [inline src/views/edm/EDMBenchmarkChart.vue]: Remove any `console.log`, it only used for testing and should be removed on the production environment.

File: src/views/edm/EDMOverviewChart.vue [hunk 2]
```diff
@@ -158,6 +165,7 @@ export default {
       },
     }),
     update() {
+      console.log('Updating EDM Chart....')
```
Comment (indrarudianto) [inline src/views/edm/EDMOverviewChart.vue]: Remove `console.log`.

File: src/views/gps/baseline/GPSBaselineChart.vue [hunk 3]
```diff
@@ -198,6 +205,9 @@ export default {
       },
     }),
   },
+  update() {
+    console.log('Updating GPSBaseline Chart...')
```
Comment (indrarudianto) [inline src/views/gps/baseline/GPSBaselineChart.vue]: Remove `console.log`.

File: src/views/seismic/SeismicEnergyChart.vue [hunk 4]
```diff
@@ -128,6 +139,7 @@ export default {
       },
     }),
     update() {
+      console.log('Updating Seismic Energy Chart...')
```
Comment (indrarudianto) [inline src/views/seismic/SeismicEnergyChart.vue]: Remove `console.log`.

File: src/views/weather/babadan/PressureChart.vue [hunk 5]
```diff
@@ -58,13 +72,22 @@ export default {
...
+    this.interval = setInterval(this.update, 60000)
+  },
   methods: {
     ...mapActions({
       fetchData(dispatch) {
         return dispatch(NAMESPACE + '/' + UPDATE_METEOROLOGY)
       },
     }),
     update() {
+      console.log('Updating Pressure Chart...')
```
Comment (indrarudianto) [inline src/views/weather/babadan/PressureChart.vue]: Remove `console.log`.

File: src/views/weather/babadan/RainfallChart.vue [hunk 6]
```diff
@@ -53,13 +67,22 @@ export default {
...
+    this.interval = setInterval(this.update, 60000)
+  },
   methods: {
     ...mapActions({
       fetchData(dispatch) {
         return dispatch(NAMESPACE + '/' + UPDATE_METEOROLOGY)
       },
     }),
     update() {
+      console.log('Updating Rainfall Chart...')
```
Comment (indrarudianto) [inline src/views/weather/babadan/RainfallChart.vue]: Remove `console.log`.

File: src/views/weather/pasarbubar/PressureChart.vue [hunk 7]
```diff
@@ -60,13 +74,22 @@ export default {
...
+    this.interval = setInterval(this.update, 60000)
+  },
   methods: {
     ...mapActions({
       fetchData(dispatch) {
         return dispatch(NAMESPACE + '/' + UPDATE_METEOROLOGY)
       },
     }),
     update() {
+      console.log('Updating Pressure Chart...')
```
Comment (indrarudianto) [inline src/views/weather/pasarbubar/PressureChart.vue]: Remove `console.log`.

File: src/views/weather/pasarbubar/RainfallChart.vue [hunk 8]
```diff
@@ -55,13 +68,22 @@ export default {
...
+    this.interval = setInterval(this.update, 60000)
+  },
   methods: {
     ...mapActions({
       fetchData(dispatch) {
         return dispatch(NAMESPACE + '/' + UPDATE_METEOROLOGY)
       },
     }),
     update() {
+      console.log('Updating Rainfall Chart...')
```
Comment (indrarudianto) [inline src/views/weather/pasarbubar/RainfallChart.vue]: Remove `console.log`.

File: src/views/weather/pasarbubar/WindRoseChart.vue [hunk 9]
```diff
@@ -29,6 +29,7 @@ import {
 } from '@/components/echarts/chart-options/wind-rose'
 import { NAMESPACE } from '@/store/weather/pasarbubar/wind-rose'
 import { UPDATE_METEOROLOGY } from '@/store/weather/pasarbubar/rainfall/actions'
+//import MoreMenu from '@/components/more-menu'
```
Comment (indrarudianto) [inline src/views/weather/pasarbubar/WindRoseChart.vue]: Remove unnecessary comment

File: src/views/weather/pasarbubar/WindRoseChart.vue [hunk 10]
```diff
@@ -69,6 +70,9 @@ export default {
       return options
     },
   },
+  /*mounted() {
+    setInterval(this.update, 60000)
+  },*/
```
Comment (indrarudianto) [inline src/views/weather/pasarbubar/WindRoseChart.vue]: Remove unused code

**Review by indrarudianto — CHANGES_REQUESTED** [review]
Remove unused `console.log`.
Remove unnecessary comments.
Remove unused code.

**Comment by fajarutamaa** [comment]
Okey sir, it's done. Please check again. Thank you 👍

**Review by indrarudianto — APPROVED** [review]

**Pull request merged**